src/browser/actions.js

```diff
@@ -62,6 +104,16 @@ define((require, exports, module) => {
     localStorage[`session@${version}`] = JSON.stringify(session.toJSON());
   };
 
+  const showDashboard = (dashboardCursor, tabStripCursor) => {
```

Comment:
If weather tabs trip's state is derived from dashboards, I'd rather have a something like:

```js
const isTabsTripVisible = tabStripCursor.get('isActive') || dashboardCursor.get('isActive')
```

Instead of synchronizing two fields.